index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+	<html>
+	<head>
+		<title>Ser Duncan the Tall</title>
+		<link rel="stylesheet" type="text/css" href="styles.css">
+		<link href='https://fonts.googleapis.com/css?family=Lobster' rel='stylesheet' type='text/css'>
+	</head>
+	<body>
```

Comment:
@filippia Excellent indentation and clean usage if empty lines to separate sections of code.